guide/src/chapter/reference_guide/concepts/content_transformer.md

```diff
@@ -0,0 +1,34 @@
+# Content Transformer
```

Comment:
why are there so many empty files? like `guide/src/chapter/reference_guide/command_line_interface/render.md`

Reply:
The goal of this pr was just to copy over the existing wiki content. The actual adding and fleshing out will be done in another one (at least that was my goal).

If you object to that, i can also start with the cleanup in this pr.

Reply:
hmm, not sure. I think it's fine, but I don't like the fact that we have now two wiki's to maintain in the meantime.

Reply:
No the goal is to shutdown the github wiki after this pr lands.
Thats why i migrated the content.

Reply:
Yeah but until the new wiki is done we have to two wiki's.

Reply:
my point is : we should move everything out of the old wiki and delete It, as soon as this PR that should include all the content from the old wiki, lands.

Reply:
But that is what we have here, or am i missing something?

What i did is migrate **all** content from the old wiki (github) to the new wiki (mdbook).
So now the new wiki contains everything from the old wiki and the old wiki can be shutdown.

I also added some additional pages to the new wiki as a future plan to work/expand on but this will be totally new content.
(This was done so that also others who want to contribute have a guideline on what can/should be documented).